daemon/lxd.go

```diff
@@ -166,3 +169,92 @@ func (l *LXD) DeleteContainer(hostname string) (bool, error) {
 
 	return true, nil
 }
+
+func (l *LXD) CreateContainerFile(container pfmodel.Container, fullPath string) error {
```

Comment:
please, explicit naming.. what is `CreateContainerFile`? it's not clear enough

Reply:
Done